ui/appui-react/src/appui-react/childwindow/InternalChildWindowManager.tsx

```diff
@@ -271,12 +271,25 @@ export class InternalChildWindowManager implements FrameworkChildWindows {
       return false;
     }
 
+    const nineZoneState =
+      UiFramework.frontstages.activeFrontstageDef?.nineZoneState;
+
+    if (nineZoneState === undefined) return false;
+
+    const popoutContentContainer = document.getElementById(
+      `content-container:${nineZoneState.widgets[childWindowId].activeTabId}`
+    );
+    if (popoutContentContainer === null) return false;
+
+    const contentWidth = popoutContentContainer.offsetWidth + 20;
+    const contentHeight = popoutContentContainer.offsetHeight + 20;
```

Comment:
The `open` method is public and a user can put anything in a child window (we just happen to use it ourselves directly for popouts) I think what you are trying to do can work by setting the `location.width` and `location.height` to `contentWidth` and `contentHeight` from wherever we call this method for the popouts though.

Reply:
Moved to internal method